SANY: Add `-help` CLI option

jubnzv opened the pull request:
Closes #511

lemmy commented:
Thanks for the PR!

To reduce code complexity, could you please inline the content of `help.txt` into the Java file and remove all loading code?  This keeps things in line with [TLC's CLI options](https://github.com/tlaplus/tlaplus/blob/master/tlatools/org.lamport.tlatools/src/tlc2/TLC.java#L1467).

jubnzv commented:
Thank you!

This `usageGenerator` is actually what I was looking for.

I updated the PR and added a description of the tool from https://lamport.azurewebsites.net/tla/tools.html.

lemmy commented:
Merged with https://github.com/tlaplus/tlaplus/commit/eff4e9b31354c6b661229205af3f6e2716a4a231
